api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -358,7 +358,7 @@ public ResponseEntity<Profile> submitDUCC(Integer duccSignedVersion, String init
   @Override
   public ResponseEntity<Profile> syncComplianceTrainingStatus() {
     try {
-      complianceTrainingService.syncComplianceTrainingStatusV2();
+      complianceTrainingService.syncComplianceTrainingStatus();
```

Comment:
oh good, now I don't have to push the silly `joel/rename-v2` local branch I started :-D